.github/workflows/pylint.yml

```diff
@@ -21,11 +23,19 @@ jobs:
         python get-pip.py
         pip install pylint==1.1.0
         pip install astroid==1.2
-        
+
     - name: Run Pylint Checks
       run: |
-        pylint --rcfile=.pylintrc convert2rhel/
-        
+         function interpreted_pylint() {
+            local es=0
```

Comment:
I'd also suggest using more verbose variable name. From a glance I don't don't know what this variable stands for. (e.g. `error` or `errorCode` etc...)